server/db/schema.models.ts

```diff
@@ -24,7 +24,11 @@ const dailyEntriesSchema = new mongoose.Schema({
     index: true,
     required: true,
   },
-  foodItems: [foodItemsSchema],
```

Comment:
@nicoleqi7 , This is why the tests are not passing, we should be using nested documents.